dev/constraints.h

```diff
@@ -465,6 +504,10 @@ namespace sqlite_orm {
         return {{}};
     }
 
+    /**
+     *  AUTOINCREMENT keyword. [Deprecation notice] Use `primary_key().autoincrement()` instead of using this function.
```

Comment:
Standard attribute `[[deprecated]]` is available!

Reply:
Same would be nice on `having()`.

Reply:
done